src/controllers/reports/reportController.ts

```
@@ -0,0 +1,264 @@
+
+    const validatedRequestBody = CreateReportRequestSchema.parse(req.body);
+    const { reportType, name } = validatedRequestBody;
+
+    let result;
+
+    //* Switch based on report type, although currently only disaster incident is supported
+    switch (reportType) {
+      case ReportType.DISASTER_INCIDENT:
+        // handle disaster incident report, infer type from validatedRequestBody to get the correct type
+        const disasterParams = (validatedRequestBody as any)
+          .parameters as z.infer<typeof DisasterIncidentParametersSchema>;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)